examples/example_project/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+project(CCCLDemo CUDA)
+
+# This example uses the CMake Package Manager (CPM) to simplify fetching CCCL from GitHub
+# For more information, see https://github.com/cpm-cmake/CPM.cmake
+include(cmake/CPM.cmake)
+
+# This will automatically clone CCCL from GitHub and make the exported cmake targets available
+CPMAddPackage(
+    NAME CCCL
+    GITHUB_REPOSITORY nvidia/cccl
+    GIT_TAG monorepo # TODO: Replace this when monorepo is merged and release is out
+    OPTIONS "CCCL_ENABLE_LIBCUDACXX OFF" "CCCL_ENABLE_CUB OFF" "CCCL_ENABLE_THRUST OFF" "CCCL_ENABLE_TESTING OFF"
+)
+
+# Default to building for the GPU on the current system
+if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
+  set(CMAKE_CUDA_ARCHITECTURES native)
+endif()
+
+add_executable(example example.cu)
+target_link_libraries(example PRIVATE cccl::cccl)
```

Comment:
@allisonvacanti this isn't currently actually possible, but I wanted to talk to you about if/how we could make this work.

The gist is, what target do we want to tell most people to link against to get all 3 libraries?